### PR TITLE
Adicionar CI #5

### PR DESCRIPTION
**Pull request by pepcmarques**
Hello Renzo, I've created the GitHub CI on this branch. Looks good

### CODE REVIEW

**Comment by undersfx** [comment]
Hey Paulo, nice one.

Just a tip in my point of view. A squash to merge all changes in just one commit makes the review process easier.

Here's a cool post that i use as reference: https://www.internalpointers.com/post/squash-commits-into-one-git

Regards.

**Comment by pepcmarques** [comment]
Hello Thiago,

Thank you for the tip. I will squash it next time.

Cheers

File: .github/workflows/pythonapp.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    - uses: actions/checkout@v1
+    - name: Set up Python 3.8
+      uses: actions/setup-python@v1
+      with:
+        python-version: 3.8
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install pipenv
+        pipenv install -d
```
Comment (renzon) [inline .github/workflows/pythonapp.yml]: Esse comando deveria ser  `pipenv sync -d`. O install vai tentar gerar o Pipfile.lock e não queremos isso, queremos build determinístico.

**Comment by pepcmarques** [comment]
Alteração para `pipenv sync -d` realizado...
O `squash` funcionou?

**Comment by renzon** [comment]
Ficaram 2 commits, mas vou aprovar assim.

**Pull request merged**